### DISCUSSION TITLE
Target Single .md File

### DISCUSSION THREAD

**Discussion by tomking** [BaileyJM02/markdown-to-pdf#26] [Q&A] [answered]
Hello, I see that the action takes an input directory to pull .md files from for conversion. I was wondering if it is possible to target a single .md file rather than an entire directory? I didn't see anything about this in the readme, apologies if I missed it and thank you for your time!

**Comment by BaileyJM02** [comment, marked as answer]
Hey @tomking this has just been released, better late than never 😅
